src/otx/core/data/entity/base.py

```diff
@@ -247,7 +247,7 @@ def _resize_image_info(image_info: ImageInfo, size: list[int], **kwargs) -> Imag
 
     ori_h, ori_w = image_info.ori_shape
     new_h, new_w = image_info.img_shape
-    image_info.scale_factor = (new_h / ori_h, new_w / ori_w)
+    image_info.scale_factor = (new_w / ori_w, new_h / ori_h)
```

Comment:
@vinnamkim @harimkang @jaegukhyun 
I tried to change `scale_factor` from `(h, w)` to `(w, h)` to match https://github.com/open-mmlab/mmcv/blob/v2.1.0/mmcv/transforms/processing.py#L193.
With `(h, w)`, predicted bounding boxes in `mmdet.base_dense_head` are wrongly post-processed.
Is there any conflict affecting other tasks?

Reply:
See https://github.com/openvinotoolkit/training_extensions/blob/977f99275066d1a1b895cea321053619b23305fe/src/otx/core/data/entity/base.py#L62-L67

Its definition is `(h,w)` and aligned with the other definitions, e.g., `img_shape`. What you mentioned is `mm*`'s problem. It should be handled in `src/otx/core/data/transform_libs/mm*.py`

Reply:
Yes, the definition is right and I think this way is totally right.
But just for now, matching `scale_factor` with mmx pipelines seems more important because it can affect other models implemented from mmx.
And if `scale_factor` with `h, w` is used, there are many things to be changed in model implementations.
So more discussion is required with other assigned people.

Reply:
How about leaving this now and update to `(h, w)` again when merging model part?